### ISSUE TITLE
`\beta` gets interpreted as `\bar{\eta}`

### ISSUE THREAD

**Issue by miguelbiron** [trevorcampbell/shortex#14]
I think this macro needs fine tuning

https://github.com/trevorcampbell/shortex/blob/167bb3459886ae2a9f1658518d29dd208e5fd440/shortex.sty#L612

**Comment by miguelbiron** [comment]
This is my current fix

```tex
% define set of greek letters excluding eta
\newcommand{\lowerCaseGreekLettersNoEta}{{alpha}{beta}{gamma}{delta}{epsilon}{zeta}{theta}{iota}{kappa}{lambda}{mu}{nu}

...

%\texalphabetmacro{b}{\boldorbar}{\lowerCaseGreekLetters} % old line
\texalphabetmacro{b}{\boldorbar}{\lowerCaseGreekLettersNoEta} % new line
\newcommand{\bareta}{\boldorbar{\eta}} % or whatever other name makes more sense
```

**Comment by jnegrea** [comment]
this is resolved by #15
